frontend/src/components/TransactionRows/MemoRow.vue

```diff
@@ -0,0 +1,23 @@
+<template>
+  <div class="memo-row">
+    <b-row>
+      <b-col cols="5">
+        <div class="text-right">{{ $t('form.memo') }}</div>
+      </b-col>
+      <b-col cols="7">
+        <div class="gdd-transaction-list-message">{{ memo }}</div>
+      </b-col>
+    </b-row>
+  </div>
+</template>
+<script>
+export default {
+  name: 'Memo-Row',
```

Comment:
Ist `-`(kebab-case)  oder nicht doch `MemoRow`(CamelCase)?

Reply:
in camelcase geändert